index.html

```diff
@@ -0,0 +1,319 @@
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                ssid_chart
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">Exhibition</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                dns
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">Platforum</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                work
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">Workshop</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col prgrmCol" id="prgrmCol2">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                ac_unit
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">ESSIgnite</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] I noticed here you don't have the button as shown in the original design so kindly add it and make its styles in the desktop version and in the mobile version since it has different styles 


- Your design

![image](https://user-images.githubusercontent.com/58553711/205466669-84f53b9a-7fb1-4092-82ed-2c5a2ba260ff.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/205466680-b3941523-07de-4756-a77b-e6ebfc857ff3.png)

- Mobile version
![image](https://user-images.githubusercontent.com/58553711/205466703-54e53a48-7031-4773-a294-aa5681585c80.png)

Reply:
Thanks for the suggestion. Now both Image and underline text is added.